readme.md

```diff
@@ -1,142 +1,30 @@
-# Lumicks pylake 
+<img src="./docs/pylake_logo_light.png#gh-light-mode-only" alt="logo" width="400px"/>
+<img src="./docs/pylake_logo_dark.png#gh-dark-mode-only" alt="logo" width="400px"/>
 
 [![DOI](https://zenodo.org/badge/133832492.svg)](https://zenodo.org/badge/latestdoi/133832492)
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](license.md)
 ![Build Status](https://github.com/lumicks/pylake/workflows/pytest/badge.svg)
 [![Documentation Status](https://readthedocs.org/projects/lumicks-pylake/badge/?version=latest)](https://lumicks-pylake.readthedocs.io/en/latest/?badge=latest)
 
-This Python package includes data analysis tools for Bluelake HDF5 data.
+Pylake is a Python package for analyzing single molecule optical tweezer data.
 
-## Install
+Its main features include:
 
-For general use, all you need to do is enter the following on the command line:
+- **Analyzing HDF5 and TIFF data obtained with Bluelake**: Pylake provides a convenient interface to read, analyze and plot data obtained with Bluelake.
+- **Correlating data**: Provides functionality to correlate optical tweezer and imaging data.
+- **Fitting force extension curves**: Pylake provides tools to (globally) fit polymer models to force extension data.  
+- **Force calibration routines**: The package contains algorithms to calibrate optical tweezer data using power spectral density estimation.
+- **Kymotracker**: Tracks single molecules in a kymograph and computes binding rates and diffusion constants.
 
-```bash
-pip install lumicks.pylake
-```
+Please see the [documentation](https://lumicks-pylake.readthedocs.io/en/latest) for more information.
```

Comment:
It's best to use the base URL because then the site controls the defaults branch. We also usually don't want to confuse users with unreleased features on the `latest` docs.
```suggestion
Please see the [documentation](https://lumicks-pylake.readthedocs.io/) for more information.
```

Reply:
Good point. I also changed it in the badge linkout.